### ISSUE TITLE
[G3000 v0.3.1] MFD Blinking Waypoints

### ISSUE THREAD

**Issue by sfdeham** [Working-Title-MSFS-Mods/fspackages#534]
**How to Reproduce**
- Start flight at LOWI by loading flightplan with departure MOGTI3H (see below)

**Expected Behavior**
- The initial waypoints (WI*) are displayed correctly

**Actual Behavior**
- The waypoints are blinking (applies only to WI*, all including MOGTI any beyond are OK)

**Remarks**
- See video at https://www.youtube.com/watch?v=gYnDmMjY4qY

**Flightplan**
```
<?xml version="1.0" encoding="UTF-8"?>

<SimBase.Document Type="AceXML" version="1,0">
    <Descr>AceXML Document</Descr>
    <FlightPlan.FlightPlan>
        <Title>LOWI to LSGS</Title>
        <FPType>IFR</FPType>
        <RouteType>HighAlt</RouteType>
        <CruisingAlt>30000.000</CruisingAlt>
        <DepartureID>LOWI</DepartureID>
        <DepartureLLA>N47° 15' 37.00",E11° 20' 38.00",+001907.00</DepartureLLA>
        <DestinationID>LSGS</DestinationID>
        <DestinationLLA>N46° 13' 9.00",E7° 19' 37.00",+001582.00</DestinationLLA>
        <Descr>LOWI to LSGS created by SimBrief</Descr>
        <DepartureName>Innsbruck</DepartureName>
        <DestinationName>Sion</DestinationName>
        <AppVersion>
            <AppVersionMajor>11</AppVersionMajor>
            <AppVersionBuild>282174</AppVersionBuild>
        </AppVersion>
        <ATCWaypoint id="LOWI">
            <ATCWaypointType>Airport</ATCWaypointType>
            <WorldPosition>N47° 15' 37.00",E11° 20' 38.00",+001907.00</WorldPosition>
            <RunwayNumberFP>26</RunwayNumberFP>
            <ICAO>
                <ICAOIdent>LOWI</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="WI505">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 15' 8.72",E11° 16' 6.85",+006100.00</WorldPosition>
            <DepartureFP>MOGT3H</DepartureFP>
            <RunwayNumberFP>26</RunwayNumberFP>
            <ICAO>
                <ICAORegion>LO</ICAORegion>
                <ICAOIdent>WI505</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="WI506">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 17' 24.69",E11° 8' 21.27",+012700.00</WorldPosition>
            <DepartureFP>MOGT3H</DepartureFP>
            <RunwayNumberFP>26</RunwayNumberFP>
            <ICAO>
                <ICAORegion>LO</ICAORegion>
                <ICAOIdent>WI506</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="WI507">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 18' 20.40",E11° 5' 9.75",+014300.00</WorldPosition>
            <DepartureFP>MOGT3H</DepartureFP>
            <RunwayNumberFP>26</RunwayNumberFP>
            <ICAO>
                <ICAORegion>LO</ICAORegion>
                <ICAOIdent>WI507</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="WI802">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 17' 46.91",E10° 50' 22.55",+019700.00</WorldPosition>
            <DepartureFP>MOGT3H</DepartureFP>
            <RunwayNumberFP>26</RunwayNumberFP>
            <ICAO>
                <ICAORegion>LO</ICAORegion>
                <ICAOIdent>WI802</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="MOGTI">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 23' 20.33",E10° 43' 0.61",+022400.00</WorldPosition>
            <DepartureFP>MOGT3H</DepartureFP>
            <RunwayNumberFP>26</RunwayNumberFP>
            <ICAO>
                <ICAORegion>LO</ICAORegion>
                <ICAOIdent>MOGTI</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="NESES">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 32' 52.56",E10° 33' 16.96",+025800.00</WorldPosition>
            <ATCAirway>P66</ATCAirway>
            <ICAO>
                <ICAORegion>LO</ICAORegion>
                <ICAOIdent>NESES</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="KPT">
            <ATCWaypointType>VOR</ATCWaypointType>
            <WorldPosition>N47° 44' 44.79",E10° 20' 59.40",+028800.00</WorldPosition>
            <ATCAirway>P66</ATCAirway>
            <ICAO>
                <ICAORegion>ED</ICAORegion>
                <ICAOIdent>KPT</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="RAVED">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 43' 45.00",E9° 40' 10.00",+030000.00</WorldPosition>
            <ATCAirway>L856</ATCAirway>
            <ICAO>
                <ICAORegion>ED</ICAORegion>
                <ICAOIdent>RAVED</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="ETOXU">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 43' 33.00",E9° 33' 2.00",+030000.00</WorldPosition>
            <ATCAirway>L856</ATCAirway>
            <ICAO>
                <ICAORegion>ED</ICAORegion>
                <ICAOIdent>ETOXU</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="NEGRA">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 43' 20.00",E9° 25' 38.00",+030000.00</WorldPosition>
            <ATCAirway>L856</ATCAirway>
            <ICAO>
                <ICAORegion>ED</ICAORegion>
                <ICAOIdent>NEGRA</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="AKABI">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 43' 1.00",E9° 14' 0.00",+030000.00</WorldPosition>
            <ATCAirway>L856</ATCAirway>
            <ICAO>
                <ICAORegion>LS</ICAORegion>
                <ICAOIdent>AKABI</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="TRA">
            <ATCWaypointType>VOR</ATCWaypointType>
            <WorldPosition>N47° 41' 22.20",E8° 26' 13.10",+030000.00</WorldPosition>
            <ATCAirway>UZ613</ATCAirway>
            <ICAO>
                <ICAORegion>LS</ICAORegion>
                <ICAOIdent>TRA</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="OLBEN">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 18' 16.00",E7° 37' 46.00",+030000.00</WorldPosition>
            <ATCAirway>Z69</ATCAirway>
            <ICAO>
                <ICAORegion>LS</ICAORegion>
                <ICAOIdent>OLBEN</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="LUTIX">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 9' 54.00",E7° 22' 14.00",+030000.00</WorldPosition>
            <ATCAirway>UN869</ATCAirway>
            <ICAO>
                <ICAORegion>LS</ICAORegion>
                <ICAOIdent>LUTIX</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="BENOT">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N47° 3' 27.70",E7° 10' 22.10",+030000.00</WorldPosition>
            <ATCAirway>UN869</ATCAirway>
            <ICAO>
                <ICAORegion>LS</ICAORegion>
                <ICAOIdent>BENOT</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="NEMOS">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N46° 54' 43.00",E6° 54' 23.60",+028700.00</WorldPosition>
            <ATCAirway>UN869</ATCAirway>
            <ICAO>
                <ICAORegion>LS</ICAORegion>
                <ICAOIdent>NEMOS</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="VADAR">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N46° 39' 26.00",E6° 45' 13.00",+021200.00</WorldPosition>
            <ATCAirway>Y58</ATCAirway>
            <ICAO>
                <ICAORegion>LS</ICAORegion>
                <ICAOIdent>VADAR</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="SOSAL">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N46° 33' 29.00",E6° 53' 4.00",+017600.00</WorldPosition>
            <ArrivalFP>VADA2N</ArrivalFP>
            <RunwayNumberFP>25</RunwayNumberFP>
            <ICAO>
                <ICAORegion>LS</ICAORegion>
                <ICAOIdent>SOSAL</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="GRANA">
            <ATCWaypointType>Intersection</ATCWaypointType>
            <WorldPosition>N46° 17' 0.50",E7° 31' 56.60",+004100.00</WorldPosition>
            <ArrivalFP>VADA2N</ArrivalFP>
            <RunwayNumberFP>25</RunwayNumberFP>
            <ICAO>
                <ICAORegion>LS</ICAORegion>
                <ICAOIdent>GRANA</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
        <ATCWaypoint id="LSGS">
            <ATCWaypointType>Airport</ATCWaypointType>
            <WorldPosition>N46° 13' 9.00",E7° 19' 37.00",+001582.00</WorldPosition>
            <RunwayNumberFP>25</RunwayNumberFP>
            <ICAO>
                <ICAOIdent>LSGS</ICAOIdent>
            </ICAO>
        </ATCWaypoint>
    </FlightPlan.FlightPlan>
</SimBase.Document>
```

**Comment by PM3Delta** [comment]
Can confirm blinking waypoint labels out of EDDF, CIND2S departure. e.g. Waypoint DF167 was blinking (but one can live with it).

**Comment by cywang90** [comment]
Fixed with 0.4.0
